.changeset/real-bees-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Adding in component type chip to search results for clarity
```

Comment:
I'm not sure component is right, looking at your screen shot and playing around on the uffizzi environment this would work for Domains, APIs, etc. as well. My first thought would be to say `spec.type` but looking at the code I'm not as sure 🤔 https://github.com/backstage/backstage/blob/62f51368cada79bf7a7facfbb09bfcaab037b429/plugins/search-common/src/types.ts#L178

Reply:
Overthinking this, it's `spec.type` so please use that 👍

Reply:
Ah yes, I def wrote that thinking component as `part of the software catalog` and not component as a proper term. Will update